client/src/main/java/tds/config/ClientSystemFlag.java

```diff
@@ -45,6 +59,9 @@ public void setIsPracticeTest(Boolean practiceTest) {
         isPracticeTest = practiceTest;
     }
 
+    /**
+     * @return {@code True} if the {@link ClientSystemFlag} is enabled; otherwise {@code False}.
+     */
     public Boolean getIsOn() {
```

Comment:
Turns out this can't be null in the database, so changed to `boolean`.  Went through all database entities and adjusted their datatypes to be primitives where appropriate (i.e. where the value can't be null in the database).